app/src/main/java/com/myperishableplanner/v21001/service/IItemService.kt

```diff
@@ -0,0 +1,4 @@
+package com.myperishableplanner.v21001.service
+
+interface IItemService {
+}
```

Comment:
Created an ItemService Interface.

Reply:
An interface should have functions.  Otherwise it is of little use, unless it is a marker interface.